packages/ui/src/index.tsx

```diff
@@ -38,7 +38,6 @@ export {
   ActionSheetCancel,
   Swiper,
   SwiperItem,
-  LinearGradient,
```

Comment:
this still needs to exist, just under native instead of core

Reply:
@peterpme Gotcha.